Standards/scs-xxxx-v1-achieving-certification.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
+
+2. The operator MUST include the official SCS compliance test suite (which does not require admin privileges) in their continuous test infrastructure (e.g., Zuul). The tests MUST be run at least nightly. For public clouds, it is recommended to offer the SCS project access to the infrastructure (ideally, including a Zuul nodepool) so the test suite runs can be triggered continuously by the SCS team. Alternatively, and for non-public clouds, the results (log files) MUST be submitted to SCS by a mechanism of SCS' choice and need to be reproduced again on request by SCS.
```

Comment:
Imho it should be discussed, whether tests should run on a daily basis. Some of them might have more impact to a deployment then other. (I am thinking about the domain manager role tests by @markus-hentsch ). For IaaS I see the necessity to test whenever an update is done in the deployment. For KaaS that might be different.

Reply:
Good point. We should partition the tests into quick ones (which should run nightly) and less quick ones.

However, I'm not sure if I understand what you mean by an update in the deployment. Someone can add a new flavor ad hoc without triggering any tests, right? And in the past, we had cases where suddenly a new flavor (that violated the specs) appeared in some cloud and then the nightly failed.

Reply:
In SIG meeting on 19.10.23, we decided to group tests into different buckets (such as light, medium, heavy) This should be reflected here.

Reply:
I added the classification. I will have to update scs-0003 as well (mainly to document the additional field in the yaml, but without repeating the intervals).